test/test_misc.py

```diff
@@ -16,6 +17,13 @@ def test_common_number_is_deprecated(self):
         self.assertEqual(len(warns), 1)
         self.assertEqual(DeprecationWarning, warns[0].category)
 
+    def test_yajl2_c_loadable(self):
+        spec = importlib.util.find_spec("ijson.backends._yajl2")
```

Comment:
I know this is completely out of date, but could you skip the test for python 2.7, since this functionality was only introduced in 3.4? I'm definitely planning to stop supporting 2.7 formally soon, but that entails some proper cleanups here and there that will take some time. Until then I'd rather ensure all tests still pass there too.

Reply:
Done. Sorry, it didn't even occur to me to look for Python 2 compatibility.